tests/integration/kubernetes/gha-run.sh

```diff
@@ -47,8 +47,10 @@ function create_cluster() {
 }
 
 function install_bats() {
-    sudo apt-get update
-    sudo apt-get -y install bats
+    sudo apt install -y software-properties-common
```

Comment:
@UnmeshDeodhar, just for the sake of helping my really bad memory, would you mind adding a comment saying we're installing `bats` from `lunar`  because we need a newer version that supports `setup_file` and `teardown_file`?